go.mod

```diff
@@ -11,28 +11,28 @@ replace (
 	github.com/prometheus/client_golang => github.com/prometheus/client_golang v0.9.3
 	golang.org/x/sys => golang.org/x/sys v0.0.0-20190830141801-acfa387b8d69
 
-	k8s.io/api => k8s.io/api v0.18.8
-	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.18.8
-	k8s.io/apimachinery => k8s.io/apimachinery v0.18.8
-	k8s.io/apiserver => k8s.io/apiserver v0.18.8
-	k8s.io/cli-runtime => k8s.io/cli-runtime v0.18.8
+	k8s.io/api => k8s.io/api v0.18.18
```

Comment:
Same question here, no breaking changes right ?

Reply:
Correct. There was an inadvertent breaking change in v1.18.15-v1.18.17, but was reverted in v1.18.18.